woocommerce-payments.php

```diff
@@ -188,3 +198,53 @@ function wcpay_show_old_jetpack_notice() {
 	</div>
 	<?php
 }
+
+/**
+ * Get custom texts for Jetpack Identity Crisis (IDC) module.
+ *
+ * @return array
+ */
+function wcpay_get_jetpack_idc_custom_content(): array {
+	$custom_content = [
+		'headerText'            => __( 'Safe Mode', 'woocommerce-payments' ),
+		'mainTitle'             => __( 'Safe Mode activated', 'woocommerce-payments' ),
+		'mainBodyText'          => __( 'We’ve detected that you have duplicate sites connected to WooCommerce Payments. When Safe Mode is active, payments will not be interupted, however some features may not be available until you’ve resolved this issue below. Safe Mode is most frequently activated when you’re transfering your site from one domain to another, or creating a staging site for testing. <safeModeLink>Learn more</safeModeLink>', 'woocommerce-payments' ),
+		'migratedTitle'         => __( 'WooCommerce Payments connection successfully transfered', 'woocommerce-payments' ),
+		'migratedBodyText'      => __( 'Safe Mode has been deactivated and WooCommerce Payments is fully functional.', 'woocommerce-payments' ),
+		'migrateCardTitle'      => __( 'Transfer connection', 'woocommerce-payments' ),
+		'migrateButtonLabel'    => __( 'Transfer your connection', 'woocommerce-payments' ),
+		'startFreshCardTitle'   => __( 'Create a new connection', 'woocommerce-payments' ),
+		'startFreshButtonLabel' => __( 'Create a new connection', 'woocommerce-payments' ),
+		'nonAdminTitle'         => __( 'Safe Mode activated', 'woocommerce-payments' ),
+		'nonAdminBodyText'      => __( 'We’ve detected that you have duplicate sites connected to WooCommerce Payments. When Safe Mode is active, payments will not be interupted, however some features may not be available until you’ve resolved this issue below. Safe Mode is most frequently activated when you’re transfering your site from one domain to another, or creating a staging site for testing. A site adminstrator can resolve this issue. <safeModeLink>Learn more</safeModeLink>', 'woocommerce-payments' ),
+		'supportURL'            => __( 'https://woocommerce.com/document/payments/faq/safe-mode/', 'woocommerce-payments' ),
```

Comment:
This is a new doc and will be published by the time of the release including this PR. 

See 2b6c0-pb